pkg/pluginhelper/status.go

```diff
@@ -0,0 +1,46 @@
+package pluginhelper
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/cloudnative-pg/cnpg-i/pkg/operator"
```

Comment:
We should add an empty line here, in order to separate standard library packages from our ones.
More than this, we add a workflow running linters inside CI.